Restaurant.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import RestaurantModel from './RestaurantModel'
+
+class Restaurant extends React.Component {
+  constructor(props) {
+    super(props)
+    this.displayName = "Restaurant"
+
+    this.model = props.model
+
+    var foodTypeImage = "";
+    if (this.model.foodType === "Burger") {
+      foodTypeImage = "assets/img/Burger.png"
+    } else {
+      foodTypeImage = "http://placeponi.es/48/48"
+    }
+
+    this.state = {
+      foodTypeImage: foodTypeImage
+    }
+  }
+
+  render() {
+    var ratings = [];
+    for (var i=0; i < this.model.rating; i++) {
```

Comment:
@RonaTross I agree. Sadly it's an integer, can't find a good alternative besides using underscore.js but it doesn't seem worth it. Any syntax alternatives in the form of iterators / other js stuff? I'd be glad to learn.